pipelines/treatment/bilhetagem_processos_manuais/tasks.py

```
@@ -0,0 +1,106 @@
+    )
+    query = f"""
+        select
+            table_id,
+            format_datetime('%Y-%m-%d %H:%M:%S', timestamp_captura) as timestamp_captura
+        from
+            {project_id}.{dataset_id}.{jae_constants.RESULTADO_VERIFICACAO_CAPTURA_TABLE_ID.value}
+        where
+            not indicador_captura_correta
+            and table_id in ({', '.join([f"'{t}'" for t in table_ids])})
+            {date_filter}
+        """
```

Ruff (0.13.1)

Possible SQL injection vector through string-based query construction

(S608)